helix-term/Cargo.toml

```diff
@@ -30,6 +30,8 @@ helix-dap = { version = "0.6", path = "../helix-dap" }
 anyhow = "1"
 once_cell = "1.9"
 
+which = "4.2"
```

Comment:
Since we're using which to validate binaries, let's also use `which` to resolve the LSP executable fully. There was some issues before with resolution on Windows if the binary is a `.cmd` file: https://github.com/helix-editor/helix/discussions/962#discussioncomment-1579516

Reply:
(did so in master already)